website/docs/r/cosmosdb_account.html.markdown

```diff
@@ -106,6 +106,16 @@ The following arguments are supported:
 
 * `enable_multiple_write_locations` - (Optional) Enable multi-master support for this Cosmos DB account.
 
+* `key_based_meta_write_access_enabled` - (Optional) Is write operations on metadata resources (databases, containers, throughput) via account keys enabled? Defaults to `true`.
+
+* `mongo_server_version` - (Optional) The Server Version of a MongoDB account. Possible values are `4.0`, `3.6`, and `3.2`. Changing this forces a new resource to be created.
```

Comment:
since the default version was 3.2 before this, would it make sense to default it to 3.2 to be consistent with the previous version and remove that breaking change? and then in 3.0 we can change it to 3.6?

also would that mean the mogo index _id is no longer required if we use 3.2 here? also removing that breaking change?

Reply:
Since the breaking change has happened for several months, I think the users have all changed their scripts. (Since it's a breaking change from service side). Do we need to break that back?

Reply:
didn't we only update the API now, so 2.57 is when we would start defaulting to 3.6, however if we default this to 3.2 it means there effectively would be no change?

Reply:
No. We only update the tests in 2.57 to adapt to the service changes. The changes have happened for several months.